### PR TITLE
Fix Navidrome connector

### PR DESCRIPTION
**Pull request by KucharczykL**
**Describe the changes you made**
I have updated the Navidrome connector to work again. It was broken because of removal of `.songInfo`.
For more information (and a better long-term solution), see https://github.com/navidrome/navidrome/issues/1364

**Additional context**
Add any other context or screenshots here.

### CODE REVIEW

**Comment by KucharczykL** [comment]
I am aware how ugly the current solution is :( Any suggestions welcome.

**Comment by rhamzeh** [comment]
@KucharczykL I sent a PR to your branch with the needed changes so that this can get merged

**Comment by KucharczykL** [comment]
@rhamzeh Ah, thank you! I seem to have forgotten about this :)
@inverse It should be ready for your review now.

File: src/connectors/navidrome.js [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+			document.querySelector(className).src);
+}
+
+Connector.getTrackArt = () => {
+	// desktop view mode
+	return getAlbumImageFromClass('.img-content') ||
+		// mobile overlay mode
+		getAlbumImageFromClass('.cover') ||
+		// mobile mini controller mode
+		getAlbumImageFromClass('.music-player-controller');
```
Comment (rhamzeh) [inline src/connectors/navidrome.js:31]:
```suggestion
		getAlbumImageFromClass('.music-player-controller', true);
```
@KucharczykL sorry I missed a flag 🙈

**Review by inverse — APPROVED** [review]

**Comment by inverse** [comment]
Will be in the next release 👍🏼

**Pull request merged**